SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.16.x   | :white_check_mark: |
+| 5.15.x   | :white_check_mark: |
+| < 5.15.x | :x:                |
```

Comment:
This seems to be in disagreement with what has been discussed and agreed long-ago, and so feels potentially misleading at this point. Certainly worthy of discussion before merging a 'policy' PR after 4 minutes at a time most folks arent around and likely just before announcing a new release that seems to disagree with it.

5.15.15 was discussed, agreed, and finally announced to be the last 5.15.x release over 9 months ago. 5.16.3 occurred 3 months later and contained various dep updates with security fixes, 5.16.4 is about to occur over 9 months later doing the same again, with no equivalent 5.15.x update release being made or ever mentioned in a discussion.

Reply:
@gemmellr I will change 5.15.x. I just missed it. Nothing changed to what have been discussed on the mailing list already.

Reply:
Ok great. I think it is also past time it was removed from the website too, so perhaps while adding 5.16.4?

Reply:
Good idea, I will do that on website as part of the 5.16.4 update.